### ISSUE TITLE
Vulnerabilities [CVE-2022-29153] CWE-918: Server-Side Request Forgery (SSRF) and [CVE-2022-24687] CWE-400: Uncontrolled Resource Consumption ('Resource Exhaustion')

### ISSUE THREAD

**Issue by wunderbarb** [spf13/viper#1432]
### Preflight Checklist

- [X] I have searched the [issue tracker](https://www.github.com/spf13/viper/issues) for an issue that matches the one I want to file, without success.
- [X] I am not looking for support or already pursued the available [support channels](https://github.com/spf13/viper/issues/new/choose) without success.
- [X] I have checked the [troubleshooting guide](https://github.com/spf13/viper/blob/master/TROUBLESHOOTING.md) for my problem, without success.

### Viper Version

1.13.0

### Go Version

1.19.0

### Config Source

Manual set

### Format

TOML

### Code reproducing the issue

```go
#!/bin/bash
go list -json -m all | docker run --rm -i sonatypecommunity/nancy:v1.0.39 sleuth
```

### Expected Behavior

nancy reports two vulnerabilities [CVE-2022-29153] and [CVE-2022-24687].   They are coming from the dependency to github.com/hashicorp/consul/api v1.12.0.
The severity of the first one is high.

github.com/hashicorp/consul/api v1.13.0 does not have these vulnerabilities.

### Actual Behavior

nancy fails  (v1.0.39) but previous detected it also.

**Comment by sagikazarmark** [comment]
Thanks for reporting. Should be fixed in the next version.

It shouldn't be a problem though, if you don't use consul or you upgrade it in your own application.

**Comment by rcollette** [comment]
BlackDuck also reports CVE-2022-40716 for Consul and recommends updating to Consul 1.15.2

**Comment by sagikazarmark** [comment]
Should be fixed in v1.14.0
